agent/dns_test.go

```diff
@@ -6688,7 +6688,7 @@ func TestDNS_trimUDPResponse_NoTrim(t *testing.T) {
 		},
 	}
 
-	cfg := loadRuntimeConfig(t, `data_dir = "a" bind_addr = "127.0.0.1"`)
+	cfg := loadRuntimeConfig(t, `node_name = "test" data_dir = "a" bind_addr = "127.0.0.1"`)
```

Comment:
stray changes?

Reply:
It was unrelated, so I put it on its own commit. This test fails locally on a mac because of how the node name is defaulted.